lib/modules/manager/bazel-module/rules.ts

```diff
@@ -0,0 +1,146 @@
+import is from '@sindresorhus/is';
+import parseGithubUrl from 'github-url-from-git';
+import { z } from 'zod';
+import { logger } from '../../../logger';
+import type { SkipReason } from '../../../types';
+import { regEx } from '../../../util/regex';
+import { BazelDatasource } from '../../datasource/bazel';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import type { PackageDependency } from '../types';
+import { RecordFragmentSchema, StringFragmentSchema } from './fragments';
+
+// Rule Schemas
+
+export interface BasePackageDep extends PackageDependency {
+  depType: string;
+  depName: string;
```

Comment:
aren't those already on base type? 🤔

Reply:
They are. However, they are optional in the base type. This makes them required. Since this code creates the more restrictive type, it does not need to perform the check for a value.